spinetoolbox/widgets/conda_envs.py

```diff
@@ -0,0 +1,103 @@
+######################################################################################################################
+# Copyright (C) 2017-2021 Spine project consortium
+# This file is part of Spine Toolbox.
+# Spine Toolbox is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General
+# Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option)
+# any later version. This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General
+# Public License for more details. You should have received a copy of the GNU Lesser General Public License along with
+# this program. If not, see <http://www.gnu.org/licenses/>.
+######################################################################################################################
+
+"""
+Conda environment widget.
+
+:author: P. Savolainen (VTT)
+:date:   19.3.2021
+"""
+from PySide2.QtWidgets import QWidget
+from PySide2.QtCore import Slot, Qt
+from PySide2.QtGui import QStandardItemModel, QStandardItem
+from spine_engine.execution_managers.conda_kernel_spec_manager import CondaKernelSpecManager
+# import environment_kernels
+# from environment_kernels.envs_conda import get_conda_env_data
+
+
+class CondaEnv(QWidget):
+    """Class for a Conda environment editor."""
+
+    def __init__(self, parent):
+        """
+
+        Args:
+            parent (QWidget): Parent widget (Settings widget)
+        """
+        from ..ui import conda_env_window  # pylint: disable=import-outside-toplevel
+
+        super().__init__(parent=parent)  # Inherits stylesheet from SettingsWindow
+        self.setWindowFlags(Qt.Window)
+        self._parent = parent  # QSettingsWidget
+        self.ui = conda_env_window.Ui_Form()
+        self.ui.setupUi(self)
+        self.setAttribute(Qt.WA_DeleteOnClose)
+        self.connect_signals()
+        self.ksm = CondaKernelSpecManager()
+        self.model = QStandardItemModel()
+        self.ui.listView.setModel(self.model)
+        all_specs = self.ksm._all_specs()
+        self.populate_model(all_specs.keys())
+        # self.ksm.conda_env_dirs.append("~/.julia/conda/3/envs/")
```

Comment:
Commented-out code?

Reply:
conda_envs.py file was not used so I removed it.